[enterprise-4.16] OSDOCS-6182: add RN entry (#75640)
openshift/openshift-docs: release_notes/ocp-4-16-release-notes.adoc

With this release, migration from the OpenShift SDN network plugin to OVN-Kubernetes is now supported on Nutanix platforms. For more information, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#nw-ovn-kubernetes-migration-about_migrate-from-openshift-sdn[Migration to the OVN-Kubernetes network plugin].

[id="ocp-4-16-coredns-improved-egress-firewall"]
==== Improved integration between CoreDNS and egress firewall (Technology Preview)

> With this release, OVN-Kubernetes uses a new `DNSNameResolver` custom resource to keep track of DNS records in your egress firewall rules, and is available as a Technology Preview. This custom resource supports this use of both wildcard DNS names and regular DNS names and allows access to DNS names regardless if the IP addresses associated with it change.

Reviewer: ```suggestion
With this release, OVN-Kubernetes uses a new `DNSNameResolver` custom resource to keep track of DNS records in your egress firewall rules, and is available as a Technology Preview. This custom resource supports the use of both wildcard DNS names and regular DNS names and allows access to DNS names regardless of the IP addresses associated with its change.
```
